depends/packages/qt.mk

```diff
@@ -133,6 +133,7 @@ define $(package)_preprocess_cmds
   patch -p1 -i $($(package)_patch_dir)/fix_qt_pkgconfig.patch &&\
   patch -p1 -i $($(package)_patch_dir)/fix_configure_mac.patch &&\
   patch -p1 -i $($(package)_patch_dir)/fix_no_printer.patch &&\
+  patch -p1 -i $($(package)_patch_dir)/fix_rcc_determinism.patch &&\
```

Comment:
We need to export the variable here as well so that non-gitian builds get stable results. See below where QT_RCC_TEST is set to 1.

Reply:
Noted, will add this soon

Reply:
swapped out the build_env variable further up in this file and didn't notice any break in gitian results determinism.